src/tests/ftest/util/mpio_utils.py

```diff
@@ -88,13 +88,7 @@ def run_romio(self, basepath, hostlist, romio_test_repo):
             ssh.load_system_host_keys()
             ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
             ssh.connect(hostlist[0])
-            _ssh_stdin, ssh_stdout, ssh_stderr = (
-                ssh.exec_command(env_variables[0] + " && " +
-                                 env_variables[1] + " && " +
-                                 env_variables[2] + " && " +
-                                 env_variables[3] + " && " +
-                                 env_variables[4] + " && " +
-                                 run_cmd))
+            _, ssh_stdout, ssh_stderr = ssh.exec_command(run_cmd)
```

Comment:
Out of curiosity, is paramiko required here?

Reply:
This is the way the test was originally written and is actually running the tests the way they (IMO) should be, which is to run them remotely on a designated test node and not on the test runner itself (which should just be a VM).  This is the model that all tests should follow, again, IMO.